app/serializers/comment_serializer.rb

```diff
@@ -0,0 +1,11 @@
+class CommentSerializer < ActiveModel::Serializer
+  include TagHelper
+
+  attributes :id, :content, :markdown, :commentable_type, :commentable_id, :owner_id, :created_at, :updated_at, :team_id
```

Comment:
should we be returning as much information as possible? like include the commentable, owner and team?